duneggd/Config/SAND_STT/STT_Backup.cfg

```diff
@@ -0,0 +1,53 @@
+[STT_Backup]
+
+#**********************************|  SAND inner vol |********************************************
+
+class                   = duneggd.SubDetector.STT_Backup.STTBuilder
+STTconfiguration        = "STT_Backup"
+nBarrelModules          = {SANDINNERVOLUME:nBarrelModules}
+halfDimension           = {SANDINNERVOLUME:halfDimension}
+Material                = {SANDINNERVOLUME:Material}
+configuration           = {SANDINNERVOLUME:configuration}
+GRAINThickness          = {SANDINNERVOLUME:GRAINThickness}
+clearenceECALGRAIN      = {SANDINNERVOLUME:clearenceECALGRAIN}
+clearenceGRAINTracker   = {SANDINNERVOLUME:clearenceGRAINTracker} # this should be read from SANDINNERVOLUME
+clearenceTrackerECAL    = {SANDINNERVOLUME:clearenceTrackerECAL} # the same
+
+#**********************************|   STT tracker   |********************************************
+
+nofUpstreamTrkMod       = 1
+nofDownstreamTrkMod     = 6
+nofC3H6ModAfterCMod     = 6
+nofC3H6ModNoRadAfterCMod = 3
+
+#**********************************|   STT MODULE    |********************************************
+
+# target
+FrameThickness          = Q("10cm")
+AddGapForSlab           = Q("7cm")
+#targetThickness         = {"CMod" : Q("4mm"),    "C3H6Mod": Q("5mm"), "C3H6ModNoRad": Q("6.89mm"), "TrkMod":Q("0mm"), "TrkMod2lyr":Q("0mm")}
```

Comment:
What is with these commented out configurations? We are using version control, so unless there is a very good reason, unused code should just be removed and not be left in as comments. Especially if there are no explanation in the comments why it is commented out.

Same in multiple places in the code.

Reply:
The commented section is cleared now.